sources/httpUtils.ts

```diff
@@ -42,16 +42,17 @@ export async function fetchUrlStream(input: string | URL, init?: RequestInit) {
   if (process.env.COREPACK_ENABLE_DOWNLOAD_PROMPT === `1`) {
     console.error(`Corepack is about to download ${input}.`);
```

Comment:
Should we also remove the period at the end to avoid any confusion with this url.

Alternatively, we could wrap the url in quotes.

Reply:
Sounds reasonable. Might be easier to copy paste in some terminals?